<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialTerminal</class>
 <widget class="QWidget" name="SerialTerminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console série</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QGroupBox" name="portBox">
     <property name="title">
      <string>Ports</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QListView" name="portView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="infoBox">
             <property name="title">
              <string>Infos</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>PID</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="pidLabel">
                  <property name="text">
                   <string>None</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>VID</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="vidLabel">
                  <property name="text">
                   <string>None</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Port</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="portLabel">
                  <property name="text">
                   <string>None</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="buttonConnect">
               <property name="text">
                <string>Connexion</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonDisconnect">
               <property name="text">
                <string>Déconnexion</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonScan">
               <property name="text">
                <string>Scan</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>Effacer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="portConnectedLabel">
         <property name="text">
          <string>None</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="textOut">
       <property name="documentTitle">
        <string/>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="commandLine"/>
       </item>
       <item>
        <widget class="QToolButton" name="buttonSend">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>portBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLine</sender>
   <signal>returnPressed()</signal>
   <receiver>buttonSend</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>textOut</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
